dev-requirements.txt

```diff
@@ -1,28 +1,130 @@
+babel==2.9.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6"
+backports.entry-points-selectable==1.1.0; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "2.7"
+bagit==1.8.1; python_version >= "3.6" and python_version < "4"
+bioblend==0.16.0; python_version >= "3.6"
+bleach==4.1.0; python_version >= "3.6"
+blessed==1.19.0; python_version >= "3.6"
+boltons==21.0.0
+boto==2.49.0; python_version >= "3.6"
+cachecontrol==0.12.9; python_version >= "3.6" and python_version < "4"
+certifi==2021.10.8; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6"
+cffi==1.15.0; sys_platform == "linux" and python_version >= "3.6"
+charset-normalizer==2.0.7; python_full_version >= "3.6.0" and python_version >= "3.6"
+click==8.0.3; python_version >= "3.6"
+colorama==0.4.4; python_version >= "3.6" and python_full_version < "3.0.0" and platform_system == "Windows" and sys_platform == "win32" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6") or platform_system == "Windows" and python_version >= "3.6" and python_full_version >= "3.5.0" and sys_platform == "win32" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6")
+coloredlogs==15.0.1; python_version >= "3.6" and python_full_version < "3.0.0" and python_version < "4" or python_version >= "3.6" and python_version < "4" and python_full_version >= "3.5.0"
+commonmark==0.9.1
+coverage==6.1.1; python_version >= "3.6"
+cryptography==35.0.0; sys_platform == "linux" and python_version >= "3.6"
+cwltool==3.1.20211020155521; python_version >= "3.6" and python_version < "4"
+dataclasses==0.8; python_version >= "3.6" and python_version < "3.7"
+decorator==4.4.2; python_version >= "3.6" and python_full_version < "3.0.0" and python_version < "4" or python_version >= "3.6" and python_version < "4" and python_full_version >= "3.2.0"
+dill==0.3.4; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.1.0" and python_version >= "3.6"
+distlib==0.3.3; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0"
+docker==4.3.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.6"
+docutils==0.17.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.6"
+enlighten==1.10.1; python_version >= "3.6"
+ephemeris==0.10.7
+filelock==3.3.2; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.6"
+flake8-import-order==0.18.1
+flake8==2.5.5
```

Comment:
This seems quite an old version. When running poetry locally, it got resolved to the latest (4.0.1). How did you install poetry? It needs to be in its own virtualenv to prevent mixing its own dependencies with the one of the project.

I'll update this with my resolved version of (*)requirements.txt, if you don't mind.

Reply:
I just followed the instructions in https://python-poetry.org/docs/, but of course that'd be great.